synthtool/gcp/templates/python_samples/noxfile.py.j2

```diff
@@ -0,0 +1,136 @@
+# Copyright 2019 Google LLC
```

Comment:
Does this stay 2019? How do these work?

Reply:
😅 I'm always unsure when I've copied a file. I'll bump the files up to 2020.

Reply:
I'm not really sure either - I've heard both ways for copying it, and have no idea for what it means we generate a new version or whatever.

Reply:
If you copy, keep the original year

Reply:
+1 to keeping original year

Reply:
Checked the OSPO docs and it generally seems to say "keep the original year":
> the year the file was created in full numeric form (e.g., 2010); don't change this if you  edit, move, or copy the file.